cli/unipipe/blueprints/github-workflow.yml.js

```diff
@@ -3,7 +3,7 @@ name: Update Instances
 concurrency: osb-instances # signal that we're modifying instances
 
 env:
-  unipipe-version: v1.2.4
+  unipipe-version: v1.3.0
```

Comment:
t: Reference version from info.ts here to always reference the version that is compatible with your unipipe cli.

Same thing for `cli/unipipe/blueprints/unipipe-osb-gcloud-cloudrun.js` which currently references v1.2.0.